tests/benchmarks/multicore/idle_stm/sysbuild.cmake

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Add remote project
+ExternalZephyrProject_Add(
+  APPLICATION remote_rad
+  SOURCE_DIR ${APP_DIR}/../../power_consumption/common/remote_sleep_forever
+  BOARD ${SB_CONFIG_BOARD}/${SB_CONFIG_SOC}/cpurad
+  BOARD_REVISION ${BOARD_REVISION}
+)
+
+# ExternalZephyrProject_Add(
+#   APPLICATION remote_ppr
+#   SOURCE_DIR  ${APP_DIR}/remote
+#   BOARD ${SB_CONFIG_BOARD}/${SB_CONFIG_SOC}/cpuppr
+#   BOARD_REVISION ${BOARD_REVISION}
+# )
```

Comment:
can go

Reply:
this was just for testing. It's uncommented now.